state/tx/indexer/kv_test.go

```diff
@@ -0,0 +1,62 @@
+package indexer
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/abci/types"
+	db "github.com/tendermint/go-db"
+	"github.com/tendermint/tendermint/types"
+)
+
+func TestKVIndex(t *testing.T) {
+	indexer := &KV{store: db.NewMemDB()}
+
+	tx := types.Tx("HELLO WORLD")
+	txResult := &types.TxResult{tx, 1, abci.ResponseDeliverTx{Data: []byte{0}, Code: abci.CodeType_OK, Log: ""}}
```

Comment:
I tried `abci.NewResultOK([]byte{0}, "").(abci.ResponseDeliverTx)`, but got an error because `abci.Result` is a non-interface type.